docs/docs/api.md

```diff
@@ -701,6 +701,17 @@ Parameters:
 - `html`: The comment to post, formatted as an HTML string, OR
 - `markdown`: The comment to post, formatted as a markdown string.
 
+### `GET /v0/comments`
+
+Gets a list of comments for a contract, ordered by creation date descending.
+
+Parameters:
+
+- `contractId`: Optional. Which contract to read comments for. Either an ID or slug must be specified.
```

Comment:
Any chance of being able to get comments across all contracts instead of just one?